doc/ref_model/chapters/chapter04.md

```diff
@@ -75,13 +88,13 @@ This section describes a set of explicit NFVI capabilities and performance measu
 
 | Ref | NFVI Capability | Unit | Definition/Notes |
 |--------------------|---------------------------|--------|--------------------------------------------|
-| e.nfvi.per.cap.001 | CPU core pinning support | Yes/No | Determining if NFVI support CPU core pinning |
-| e.nfvi.per.cap.002 | NUMA support | Yes/No | Determining if NFVI support NUMA awareness |
-| e.nfvi.per.cap.003 | IPSec Acceleration | Yes/No | IPSec Acceleration |
-| e.nfvi.per.cap.004 | Crypto Acceleration | Yes/No | Crypto Acceleration |
-| e.nfvi.per.cap.005 | Transcoding Acceleration | Yes/No | Transcoding Acceleration |
-| e.nfvi.per.cap.006 | Programmable Acceleration | Yes/No | Programmable Acceleration |
-| e.nfvi.per.cap.007 | Enhanced Cache Management* | enum | L=Lean; E=Equal; X=eXpanded |
+| e.nfvi.cap.006 | CPU core pinning support | Yes/No | Indicates if NFVI supports CPU core pinning |
+| e.nfvi.cap.007 | NUMA support | Yes/No | Indicates if NFVI supports NUMA |
+| e.nfvi.cap.008 | IPSec Acceleration | Yes/No | IPSec Acceleration |
+| e.nfvi.cap.009 | Crypto Acceleration | Yes/No | Crypto Acceleration |
+| e.nfvi.cap.010 | Transcoding Acceleration | Yes/No | Transcoding Acceleration |
+| e.nfvi.cap.011 | Programmable Acceleration | Yes/No | Programmable Acceleration |
+| e.nfvi.cap.012 | Enhanced Cache Management* | Yes/No | If supported, L=Lean; E=Equal; X=eXpanded |
```

Comment:
small question: How do we express L/E/X capability with a yes/no Unit? Shouldn't it be enum with L=Lean; E=Equal; X=eXpanded and N=NotSupported?

Reply:
I'm glad I'm not the only one who thought that! That was the logic I used originally, but the datum is actually supposed to represent whether or not the capability to control the cache _is present_ in the give infra, which it either is or it is not, hence the flag is typed as a bool. When you configure it, if it's supported, you would use the enum (L/E/X). Thx, -Mark